src/coldcard.ts

```diff
@@ -550,6 +529,18 @@ export class ColdcardSignMultisigTransaction extends ColdcardInteraction {
  *
  */
 export class ColdcardMultisigWalletConfig {
+  jsonConfig: any;
```

Comment:
This is already typed actually [where it's called](https://github.com/unchained-capital/unchained-wallets/blob/master/src/index.ts#L580C33-L580C33) in index.ts
```suggestion
  jsonConfig: string | MultisigWalletConfig;
```

Reply:
Not so simple. The entire constructor must be reworked if this gets a type. See above about scope.